### ISSUE TITLE
two-word substitutions not working?

### ISSUE THREAD

**Issue by nickeubank** [knausj85/knausj_talon#507]
I've had "get hub" mapped to "github" for a while (as well as a few others, like "group by" to "groupby"), but it seems like it's stopped working (I think since substitutions moved to the `words_to_replace.csv` file). 

I've tried both putting it in `words_to_replace.csv`:

```
github,get hub
```

And back in the vocabulary.py file: 

```
_word_map_defaults = {
    # E.g:
    # "cash": "cache",
    # This is the opposite ordering to words_to_replace.csv (the latter has the target word first)
    "i": "I",
    "i'm": "I'm",
    "i've": "I've",
    "i'll": "I'll",
    "i'd": "I'd",
    "group by": "groupby",
    "get hub": "github",
}
```

But they don't seem to be working any more with `Say` or `Sentence`. 

Suggestions?

**Comment by rntz** [comment]
Long term I think we ought to fix this to support multi-word replacements, but short term a solution is to put them in additional_words.csv instead.

**Comment by nickeubank** [comment]
Ah, thanks for the heads up on vocabulary.py. 

Yes, I think it would be really nice if we could bring back support for multi-word replacements!

**Comment by rntz** [comment]
Properly fixed now by #641 :).
